client/settings/check-starcraft-path.js

```diff
@@ -42,14 +58,22 @@ async function checkHash(path, validHashes) {
 // be checked for other copies. If downgradePath contains files that match the correct hashes, this
 // will be counted as having the correct version, but `downgradePath` will be true.
 export async function checkStarcraftPath(dirPath, downgradePath) {
-  const requiredFiles = ['starcraft.exe', 'storm.dll', 'local.dll', 'stardat.mpq', 'broodat.mpq']
+  const requiredFiles = ['starcraft.exe', 'storm.dll', 'stardat.mpq', 'broodat.mpq']
 
   try {
     await Promise.all(requiredFiles.map(f => accessAsync(path.join(dirPath, f), fs.constants.R_OK)))
   } catch (err) {
     return { path: false, version: false, downgradePath: false }
   }
 
+  // Due to 1.19 version moving local.dll to a separate folder, we need to handle it separately
+  const matches = await globAsync(`${dirPath}/**/local.dll`)
+  if (matches.length > 0) {
+    maybeCopyLocalDll(matches[0], downgradePath)
```

Comment:
Hmm, I thought you said we should copy the local.dll file to our downgrade folder for all BW versions. I assumed that meant even for 1.16.1 - 1.18 where the local.dll was in the correct spot (BW's root folder). Are you saying we shouldn't copy the local.dll to our downgrade folder if it's located in the BW's root folder?